test/e2e/l7networkpolicy_test.go

```diff
@@ -132,30 +133,34 @@ func createL7NetworkPolicy(t *testing.T,
 	assert.NoError(t, err)
 }
 
-func probeL7NetworkPolicyHTTP(t *testing.T, data *TestData, serverPodName, clientPodName string, serverIPs []*net.IP, allowHTTPPathHostname, allowHTTPPathClientIP bool) {
-	for _, ip := range serverIPs {
+func probeL7NetworkPolicyHTTP(t *testing.T, data *TestData, serverPodName, clientPodName string, targetIPs []*net.IP, allowHTTPPathHostname, allowHTTPPathClientIP bool) {
+	for _, ip := range targetIPs {
 		baseURL := net.JoinHostPort(ip.String(), "8080")
 
 		// Verify that access to path /clientip is as expected.
 		assert.Eventually(t, func() bool {
-			cmd := []string{"wget", "-O", "-", fmt.Sprintf("%s/%s", baseURL, "clientip"), "-T", "1", "-t", "1"}
+			url := fmt.Sprintf("%s/%s", baseURL, "clientip")
+			cmd := []string{"wget", "-O", "-", url, "-T", "1", "-t", "1"}
 			_, _, err := data.RunCommandFromPod(data.testNamespace, clientPodName, agnhostContainerName, cmd)
 			if (allowHTTPPathClientIP && err != nil) || (!allowHTTPPathClientIP && err == nil) {
 				return false
 			}
+			t.Logf("Access to URL %s: expected=%t", url, allowHTTPPathClientIP)
```

Comment:
I am not opposed to adding more logs here, but I am not sure how helpful this one is. It doesn't seem like it belongs in the assert.Eventually condition function, as we only print it after we already know that the condition is met.

Reply:
Restored the changes.